substrate/frame/support/src/traits/tokens/nonfungible_v2.rs

```diff
@@ -60,10 +60,12 @@ pub trait Inspect<AccountId> {
 		None
 	}
 
-	/// Returns the system attribute value of `item` corresponding to `key`.
+	/// Returns the system attribute value of `item` of `collection` corresponding to `key` if
+	/// `item` is `Some`. Otherwise, returns the system attribute value of `collection`
+	/// corresponding to `key`.
 	///
 	/// By default this is `None`; no attributes are defined.
-	fn system_attribute(_item: &Self::ItemId, _key: &[u8]) -> Option<Vec<u8>> {
+	fn system_attribute(_item: Option<&Self::ItemId>, _key: &[u8]) -> Option<Vec<u8>> {
```

Comment:
This file represents the traits to work with a particular item and not a collection, which means you don't need to make the item param optional